numpyro/examples/datasets.py

```diff
@@ -53,6 +53,11 @@
 ])
 
 
+LYNXHARE = dset('lynxhare', [
+    'http://people.whitman.edu/~hundledr/courses/M250F03/LynxHare.txt',
```

Comment:
@neerajprad Could you help me upload this dataset to cloud?

Reply:
link - https://d2hg8soec8ck9v.cloudfront.net/datasets/LynxHare.txt

Reply:
Thanks, @neerajprad !